interfaces/Portalicious/src/app/domains/project/project.helper.ts

```diff
@@ -85,16 +82,68 @@ export function projectHasPhysicalCardSupport(project?: Project) {
 }
 
 export function projectHasFspWithExportFileIntegration(project?: Project) {
-  return project?.financialServiceProviders.some(
+  return project?.programFinancialServiceProviderConfigurations.some(
     (fsp) =>
-      fsp.integrationType === FinancialServiceProviderIntegrationType.csv,
+      getFinancialServiceProviderSettingByName(fsp.financialServiceProviderName)
+        ?.integrationType === FinancialServiceProviderIntegrationType.csv,
+  );
+}
+
+function getFspSettingsFromProjectFinancialServiceProviderConfigurationNames(
+  project: Project,
+  projectFinancialServiceProviderConfigurationNames: string[],
+) {
+  return projectFinancialServiceProviderConfigurationNames.map(
+    (projectFinancialServiceProviderConfigurationName) => {
+      const config = project.programFinancialServiceProviderConfigurations.find(
+        (fsp) => fsp.name === projectFinancialServiceProviderConfigurationName,
+      );
+
+      if (!config) {
+        throw new Error(
+          `Could not find financial service provider configuration with name ${projectFinancialServiceProviderConfigurationName}`,
+        );
+      }
+
+      return getFinancialServiceProviderSettingByName(
+        config.financialServiceProviderName,
+      );
+    },
   );
 }
 
-export function fspsHaveExcelFsp(fsps: FinancialServiceProviders[]) {
-  return fsps.some((fsp) => fsp === FinancialServiceProviders.excel);
+export function financialServiceProviderConfigurationNamesHaveIntegratedFsp(
```

Comment:
Maybe it makes more sense to have one function:

`financialServiceProviderConfigurationsHaveIntegrationType` that replacesfinancialServiceProviderConfigurationNamesHaveExcelFsp and financialServiceProviderConfigurationNamesHaveIntegratedFsp

that takes in integrationType as argument.  If you think this is good enough/better that is also fine for me